doc/man3/CMS_decrypt.pod

```diff
@@ -54,26 +72,32 @@ open to attack.
 
 It is possible to determine the correct recipient key by other means (for
 example looking them up in a database) and setting them in the CMS structure
-in advance using the CMS utility functions such as CMS_set1_pkey(). In this
-case both B<cert> and B<pkey> should be set to NULL.
+in advance using the CMS utility functions such as CMS_set1_pkey(),
+or use CMS_decrypt_set1_password() if the recipient has a symmetric key.
+In these cases both I<cert> and I<pkey> should be set to NULL.
 
 To process KEKRecipientInfo types CMS_set1_key() or CMS_RecipientInfo_set0_key()
 and CMS_RecipientInfo_decrypt() should be called before CMS_decrypt() and
-B<cert> and B<pkey> set to NULL.
+I<cert> and I<pkey> set to NULL.
 
-The following flags can be passed in the B<flags> parameter.
+The following flags can be passed in the I<flags> parameter.
```

Comment:
Is it worth adding a list of the flags here?  The descriptions could either be inline or later.

It seems like a list should go here but there isn't one.

Reply:
> Is it worth adding a list of the flags here? The descriptions could either be inline or later.
> 
> It seems like a list should go here but there isn't one.

I agree. Yet I do not know whether further flags apply in this case.
Here my aim was just to update the symbol layout.
The original text was given by @richsalz  in 99d63d4662e16afbeff49f29b48f1c87d5558ed0.